src/lib/git/git.ts

```diff
@@ -46,3 +47,27 @@ export async function onChangeLastCommit(
 
 	return interval;
 }
+
+export async function gitCheckout(branch: string): Promise<void> {
+	const gitAPI = getGitAPI();
+	const localRepoUri = await getLocalRepoUri();
```

Comment:
I'd say just do the same as in `src/lib/gerrit/gerrit.ts:12`, we'll just fail if they have 0 or more than 1 git repo (because who does that) and if they have 1 we just take the one registered to the git API (`gitAPI.repositories[0]`). Saves effort when it comes to adding an option and also makes it so the user doesn't have to enter a value for that option.